### ISSUE TITLE
input-name rule option to specify casing

### ISSUE THREAD

**Issue by lioneltay** [dotansimha/graphql-eslint#276]
At the moment, when using the `input-name` rule https://github.com/dotansimha/graphql-eslint/blob/master/docs/rules/input-name.md, this is an error

```graphql
# eslint @graphql-eslint/input-name: ["error", [{"checkInputType":true}]]

type Mutation {
  setMessage(input: SetMessageInput): String
}
```

While this passes

```graphql
# eslint @graphql-eslint/input-name: ["error", [{"checkInputType":true}]]

type Mutation {
  SetMessage(input: SetMessageInput): String
}
```

Could an option be added to allow the mutation name to still be in camel case, while the type name should be the PascalCase version appended with `Input`?

Maybe this will allow this rule to work better with the `naming-convention` rule https://github.com/dotansimha/graphql-eslint/blob/master/docs/rules/naming-convention.md.

**Comment by dotansimha** [comment]
Fixed in `@graphql-eslint/eslint-plugin@0.8.0`
